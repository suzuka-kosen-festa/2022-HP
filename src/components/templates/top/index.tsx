import type { FC } from "react";
import tw from "twin.macro";
import type { TopProperties } from "./types/model";
import { Caution } from "../../molecules/caution";
import { Text } from "../../atoms/text";
import { TicketSection } from "../../organisms/ticket-section";
import { FestaSection } from "../../organisms/festa-section";
import { SnsSection } from "../../organisms/sns-section";
import type { SocialData } from "../../organisms/sns-section/types/model";
import { BazaarSection } from "../../organisms/bazaar-section";
import { MapSection } from "../../organisms/map-section";
import { SponsorSection } from "../../organisms/sponsor-section";
import { ContactSection } from "../../organisms/contact-section";
import { AccessSection } from "../../organisms/access-section";
import { AboutSection } from "../../organisms/about-section";
import { PickupSection } from "../../organisms/pickup-section";

const Highlight = tw.span`text-primary-red text-base`;

const CautionText = tw(Text)`text-base sm:text-base`;

const Main = tw.main`space-y-11 sm:space-y-20`;

const CautionBox = tw.section`flex flex-col justify-center items-center`;

const cautions = [
  <CautionText key="0">30日(日)は<Highlight>午前9時より午後3時</Highlight>、31日(月)は<Highlight>午前9時より午後4時</Highlight>に参加いただけます。</CautionText>,
  <CautionText key="1">
    Re:ROADでは、感染拡大防止のため<Highlight>入場制限</Highlight>を設けています。ご入場にはチケットが必要となります。
  </CautionText>,
  <CautionText key="2">発熱や風邪などの症状がある方のご入場はご遠慮ください。</CautionText>,
  <CautionText key="3">
    イベントにおける<Highlight>声出し</Highlight>はご遠慮いただき、<Highlight>拍手</Highlight>での応援にご協力ください。
  </CautionText>,
  <CautionText key="4">駐車場はスペースが限られているため、できるだけ公共交通機関を使ってお越しください。</CautionText>,
];

const socialLinks: Array<SocialData> = [
  {
    icon: "ci:twitter",
    key: "Twitter",
    link: "https://twitter.com/KOSENFESTA",
    style: tw`text-social-twitter hover:text-social-twitter-reaction`,
  },
  {
    icon: "ci:instagram",
    key: "Instagram",
    link: "https://www.instagram.com/kosenfesta/",
    style: tw`text-white bg-social-instagram hover:bg-social-instagram-reaction rounded-md sm:rounded-2xl`,
  },
  {
    icon: "ci:youtube",
    key: "YouTube",
    link: "https://www.youtube.com/channel/UCV4zdsdagV2N3juHRz2dqQQ",
    style: tw`text-social-youtube hover:text-social-youtube-reaction`,
  },
  {
    icon: "ci:spotify",
    key: "Spotify",
    link: "https://open.spotify.com/show/4n0D0dsaFaPbJTz7tGpfTR?si=8yZ1H8LzTM-ZJPyYN6CU-A",
    style: tw`text-social-spotify hover:text-social-spotify-reaction`,
  },
];

const attentions = [
  <CautionText key="1">学内における飲食物の食べ歩きはご遠慮ください。</CautionText>,
  <CautionText key="2">学内バザーにて購入した飲食物のお持ち帰りはご遠慮ください</CautionText>,
  <CautionText key="3">
    各出店バザーにお並びいただく際、並ばれる人数が多い場合は他のブースに移っていただくようにお声がけさせていただく場合がありますので、当日は係員の指示に従ってください。
  </CautionText>,
];

const Top: FC<TopProperties> = ({ events }) => (
  <Main aria-label="Top page">
    <CautionBox aria-label="『Re:ROAD』開催にあたってのお願い">
      <Caution sentence={cautions} title="『Re:ROAD』開催にあたってのお願い" />
    </CautionBox>
    <TicketSection title="Tickets">
      {
        "入場制限により、ご入場いただける方を一部の方のみに限定させていただいております。\n締切: 10/23 (日)\n\nまた、中学生の皆様向けに 中学生受験相談室 というイベントを実施いたしますので、ぜひご参加ください。\n日時：10/30（日)　10:30~12:00、14:00~15:30\n場所：マルチメディア棟1階 図書館"
      }
    </TicketSection>
    <FestaSection subtitle="今年のテーマ『Re:ROAD』について" buttonText="第57回鈴鹿高専祭メンバー" title="About">
      {
        "第57回鈴鹿高専祭、テーマは「Re:ROAD」\n今回のテーマ「Re:ROAD」には「道（Road）」を創り直す（Re）という意味と、「弾をこめる・再補填する（Reload）」という意味を使い、「今年の準備をする」「来年以降、これから先のための高専祭を行う」という想いが込められています。"
      }
    </FestaSection>
    <SnsSection socialData={socialLinks} title="SNS">
      {"今年の鈴鹿高専祭では、史上初となるラジオの配信を行っています。\nご視聴はYouTube、Spotifyから！"}
    </SnsSection>
    <PickupSection buttonText="タイムスケジュールを見る" title="Pick up events" events={events} />
    <BazaarSection
      attentionTitle="バザーに関するお願い"
      attentions={attentions}
      buttonText="バザーの詳細を見る"
      description={
        "飲食団体は18団体、その他レクリエーションには17団体が出店しています！ \n屋台・レク等のバザー詳細についてはこちらをクリック！"
      }
      title="Bazaar"
    />
    <MapSection title="Map" />
    <SponsorSection buttonText="READ MORE" title="Sponsor">
      {
        "第57回鈴鹿高専祭を開催するにあたり、多くの方にご支援賜りました。\nこちらのページにてご協賛いただいた企業様、ご支援いただいた個人様のお名前を記載しておりますのでぜひご覧ください。"
      }
    </SponsorSection>
    <ContactSection buttonText="お問い合わせ" link="https://forms.gle/MU9338vfudKPEnwS7" title="Contact us">
      第57回鈴鹿高専祭に関するお問い合わせは下のボタンからお願いいたします（google formsが開きます）
    </ContactSection>
    <AccessSection
      attention={
        "駐車場はスペースに限りがございますのでご了承ください。\n30日(日)は鈴鹿医療科学大学 白子キャンパス 様\n31日(月)はMEGAドン・キホーテUNY鈴鹿店 様の屋上駐車場　をご利用ください。"
      }
      center={{
        lat: 34.851_671_242_413_17,
        lng: 136.581_323_097_843_55,
      }}
      zoom={15}
      label="鈴鹿工業高等専門学校"
      info={"510-0294 三重県鈴鹿市白子町\n近鉄白子駅から徒歩約30分\n東旭が丘三丁目バス停から徒歩約7分"}
      subtitle="お車でお越しの方へ"
      title="Access"
    />
    <AboutSection buttonText="鈴鹿高専ホームページ" link="https://www.suzuka-ct.ac.jp" title="NIT,Suzuka College">
      鈴鹿高専については本校ホームページをご覧ください。
    </AboutSection>
  </Main>
);

export { Top };
