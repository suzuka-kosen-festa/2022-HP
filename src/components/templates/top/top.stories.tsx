import type { ComponentStoryObj, ComponentMeta } from "@storybook/react";
import { ComponentPropsWithoutRef } from "react";
import { Top } from ".";

type T = typeof Top;
type Story = ComponentStoryObj<T>;
type Meta = ComponentMeta<T>;

const args: ComponentPropsWithoutRef<T> = {
  events: [
    {
      id: 1,
      title: "オープニング",
      descriptions: "空白の2年間、たくさんの協力を得て遂に開催！溜め込んだものを全て出し切れ！高専祭開幕です！",
      date: "2022-10-30 09:00",
      venue: "メインステージ",
      start_time: "2022-10-30 09:00",
      end_time: "2022-10-30 09:30",
      stage: "main",
    },
    {
      id: 2,
      title: "音楽部コンサート",
      descriptions: "音楽部による、メインステージ最初を飾るセレモニー！盛り上がっていきましょー！",
      date: "2022-10-30 09:30",
      venue: "メインステージ",
      start_time: "2022-10-30 09:30",
      end_time: "2022-10-30 10:00",
      stage: "main",
    },
    {
      id: 3,
      title: "常識力バトル",
      descriptions: "高専生の常識をクイズ形式で競うバトル！高専生としての常識がある学生は誰か！？",
      date: "2022-10-30 10:00",
      venue: "メインステージ",
      start_time: "2022-10-30 10:00",
      end_time: "2022-10-30 10:30",
      stage: "main",
    },
    {
      id: 4,
      title: "勝山組",
      descriptions: "今年もよさこい勝山組の参加！鈴フェス大賞のよさこいを見逃すな！",
      date: "2022-10-30 10:45",
      venue: "メインステージ",
      start_time: "2022-10-30 10:45",
      end_time: "2022-10-30 11:15",
      stage: "main",
    },
  ],
};

export default {
  args,
  argTypes: {
    events: {
      description: "Data of pick up event",
      control: {
        type: "array",
      },
    },
  },
  component: Top,
} as Meta;

export const Default: Story = {};
