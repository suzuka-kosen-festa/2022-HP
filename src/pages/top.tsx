import type { FC } from "react";
import useSWR from "swr";
import { Redirect} from "wouter";
import { Top } from "../components/templates/top";
import { fetcher } from "../libs/axios";
import type { LiveEvent} from "../api/prisma";

const TopPage: FC = () => {
  const { data, error } = useSWR<ReadonlyArray<LiveEvent>>("/liveevent/near", fetcher)
  if (!data) return <p>Loading...</p>;
  if (error) return <Redirect to="/error" />;
  return <Top events={data} />
};

export { TopPage };
