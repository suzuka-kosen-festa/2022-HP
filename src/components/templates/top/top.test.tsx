import { composeStories } from "@storybook/testing-react";
import "@testing-library/jest-dom";
import { render } from "@testing-library/react";
import * as stories from "./top.stories";

const { Default } = composeStories(stories);

const options = {
  name: "Top page",
};

describe("(components) templates/top", () => {
  test("to be templates", () => {
    const { container } = render(<Default />);
    expect(container).toBeTemplate();
  });
  test("to be [role=main]", () => {
    const { getByRole } = render(<Default />);
    expect(getByRole("main", options)).toBeInTheDocument();
  });
  test("take snap shot", () => {
    const { container } = render(<Default />);
    expect(container).toMatchSnapshot();
  });
});
