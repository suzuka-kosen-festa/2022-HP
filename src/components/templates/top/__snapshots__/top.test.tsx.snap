// Vitest Snapshot v1

exports[`(components) templates/top > take snap shot 1`] = `
<div>
  <main
    aria-label="Top page"
    class="css-1wx4hf8"
  >
    <section
      aria-label="『Re:ROAD』開催にあたってのお願い"
      class="css-wkpw2c"
    >
      <section
        aria-label="注意事項"
        class="css-u8u4n6"
        role="alert"
      >
        <div
          class="css-1m7spai"
        >
          <div
            class="css-1vlcokg"
          >
            <span />
          </div>
          <h2
            class="css-qt9rcz-Caution"
          >
            『Re:ROAD』開催にあたってのお願い
          </h2>
        </div>
        <ul
          class="css-mvofyw-Caution"
        >
          <li
            class="css-u80qbw-Caution"
          >
            <p
              class="css-17zcuf2"
            >
              Re:ROADでは、感染拡大防止のため
              <span
                class="css-786g6r"
              >
                入場制限
              </span>
              を設けています。ご入場にはチケットが必要となります。
            </p>
          </li>
          <li
            class="css-u80qbw-Caution"
          >
            <p
              class="css-17zcuf2"
            >
              発熱や風邪などの症状がある方のご入場はご遠慮ください。
            </p>
          </li>
          <li
            class="css-u80qbw-Caution"
          >
            <p
              class="css-17zcuf2"
            >
              イベントにおける
              <span
                class="css-786g6r"
              >
                声出し
              </span>
              はご遠慮いただき、
              <span
                class="css-786g6r"
              >
                拍手
              </span>
              での応援にご協力ください。
            </p>
          </li>
          <li
            class="css-u80qbw-Caution"
          >
            <p
              class="css-17zcuf2"
            >
              駐車場はスペースが限られているため、できるだけ公共交通機関を使ってお越しください。
            </p>
          </li>
        </ul>
      </section>
    </section>
    <section
      aria-label="ticket"
      class="css-wkpw2c"
    >
      <h1
        class="css-1et2tyq-TicketSection"
      >
        Tickets
      </h1>
      <p
        class="css-6o3xps-TicketSection"
      >
        入場制限により、ご入場いただける方を一部の方のみに限定させていただいております。

また、中学生の皆様向けに 中学生受験相談室 というイベントを実施いたしますので、ぜひご参加ください。
日時：10/30（日)　10:30~12:00、14:00~15:30
場所：マルチメディア棟1階 図書館
      </p>
      <nav
        class="css-1ipjmcm"
      >
        <a
          class="css-jabq07-Button"
          href="https://docs.google.com/forms/d/e/1FAIpQLSfOW-ZRsgRQFaZfdusbo_7IycPz6i0dgo2ou-TEk_UEZZIfrw/viewform"
          rel="noopener noreferrer"
          tabindex="0"
          target="_blank"
        >
          <p
            class="css-7bzdgk-Button"
          >
            中学生とその保護者の皆様の入場受付はこちら
          </p>
        </a>
        <a
          class="css-jabq07-Button"
          href="https://docs.google.com/forms/d/e/1FAIpQLSeuij0pjIOD8JTS__argba787SJ6Mj-hkpUNlDiPfo7ezrf5A/viewform"
          rel="noopener noreferrer"
          tabindex="0"
          target="_blank"
        >
          <p
            class="css-7bzdgk-Button"
          >
            OB・OGの皆様の入場受付はこちら
          </p>
        </a>
      </nav>
    </section>
    <section
      aria-label="今年のテーマ「Re:ROAD」について"
      class="css-wkpw2c"
    >
      <h1
        class="css-1uc4aan-FestaSection"
      >
        About
      </h1>
      <h2
        class="css-kv5ug9-FestaSection"
      >
        今年のテーマ『Re:ROAD』について
      </h2>
      <p
        class="css-l2da8p-FestaSection"
      >
        第57回鈴鹿高専祭、テーマは「Re:ROAD」
今回のテーマ「Re:ROAD」には「道（Road）」を創り直す（Re）という意味と、「弾をこめる・再補填する（Reload）」という意味を使い、「今年の準備をする」「来年以降、これから先のための高専祭を行う」という想いが込められています。

2019年を最後に鈴鹿高専祭は行われていません。
そのため過去のデータもあまり残っておらず本当に1から、いや0からのスタートです。
まだ何色にも色づいていない第57回鈴鹿高専祭を今から色づけていきます。
      </p>
      <a
        class="css-1eis7vz-Button"
        href="/member"
        role="button"
        tabindex="0"
      >
        <p
          class="css-oca8th-Button"
        >
          第57回鈴鹿高専祭メンバー
        </p>
      </a>
    </section>
    <section
      aria-label="SNS"
      class="css-wkpw2c"
    >
      <h1
        class="css-1sj4na4-SnsSection"
      >
        SNS
      </h1>
      <p
        class="css-1kjefg3-SnsSection"
      >
        今年の鈴鹿高専祭では、史上初となるラジオの配信を行っています。
ご視聴はYouTube、Spotifyから！
      </p>
      <div
        class="css-yczkn0"
      >
        <a
          class="css-9uxhpu-SnsSection"
          href="https://twitter.com/KOSENFESTA"
          rel="noopener noreferrer"
          target="_blank"
        >
          <span />
        </a>
        <a
          class="css-9uxhpu-SnsSection"
          href="https://www.instagram.com/kosenfesta/"
          rel="noopener noreferrer"
          target="_blank"
        >
          <span />
        </a>
        <a
          class="css-9uxhpu-SnsSection"
          href="https://www.youtube.com/channel/UCV4zdsdagV2N3juHRz2dqQQ"
          rel="noopener noreferrer"
          target="_blank"
        >
          <span />
        </a>
        <a
          class="css-9uxhpu-SnsSection"
          href="https://open.spotify.com/show/4n0D0dsaFaPbJTz7tGpfTR?si=8yZ1H8LzTM-ZJPyYN6CU-A"
          rel="noopener noreferrer"
          target="_blank"
        >
          <span />
        </a>
      </div>
    </section>
    <section
      aria-label="pick up event"
      class="css-wkpw2c"
    >
      <h1
        class="css-16hzca7-PickupSection"
      >
        Pick up events
      </h1>
      <ul
        class="css-9y1ynn"
      >
        <li
          class="css-10pxw4f"
        >
          <h2
            class="css-1a2c2qr"
          >
            へぇ王
          </h2>
          <p
            class="css-y87pn1"
          >
            先生たちが自分の研究を紹介！4年生は卒研部を考えるいい機会に！高専生、中学生に向けた勉強イベント！
          </p>
          <div
            class="css-18lwsxv"
          >
            <p
              class="css-y87pn1"
            >
              日時: 
              10/30 (日) 11:30~
            </p>
            <p
              class="css-y87pn1"
            >
              場所: 
              メインステージ
            </p>
          </div>
        </li>
        <li
          class="css-10pxw4f"
        >
          <h2
            class="css-1a2c2qr"
          >
            TTTGP
          </h2>
          <p
            class="css-y87pn1"
          >
            その名もTop of TikTok GP！TikTokで流行ったダンス、音源、TikTokerなどTikTokに関するクイズを出題！
          </p>
          <div
            class="css-18lwsxv"
          >
            <p
              class="css-y87pn1"
            >
              日時: 
              10/30 (日) 13:15~
            </p>
            <p
              class="css-y87pn1"
            >
              場所: 
              メインステージ
            </p>
          </div>
        </li>
        <li
          class="css-10pxw4f"
        >
          <h2
            class="css-1a2c2qr"
          >
            俺1GP
          </h2>
          <p
            class="css-y87pn1"
          >
            鈴鹿高専のお笑いライブ！笑いで全てを包み込め！
          </p>
          <div
            class="css-18lwsxv"
          >
            <p
              class="css-y87pn1"
            >
              日時: 
              10/31 (月) 13:00~
            </p>
            <p
              class="css-y87pn1"
            >
              場所: 
              メインステージ
            </p>
          </div>
        </li>
        <li
          class="css-10pxw4f"
        >
          <h2
            class="css-1a2c2qr"
          >
            メモリーズ
          </h2>
          <p
            class="css-y87pn1"
          >
            5年生のダンス披露！みんなで頑張って踊ります！
          </p>
          <div
            class="css-18lwsxv"
          >
            <p
              class="css-y87pn1"
            >
              日時: 
              10/31 (月) 14:15~
            </p>
            <p
              class="css-y87pn1"
            >
              場所: 
              メインステージ
            </p>
          </div>
        </li>
      </ul>
      <a
        class="css-1xkpwia-Button"
        href="/timetable"
        role="button"
        tabindex="0"
      >
        <p
          class="css-7bzdgk-Button"
        >
          タイムスケジュールを見る
        </p>
      </a>
    </section>
    <section
      aria-label="bazaar"
      class="css-wkpw2c"
    >
      <h1
        class="css-wtcl1w-BazaarSection"
      >
        Bazaar
      </h1>
      <section
        aria-label="注意事項"
        class="css-1kphsmg-BazaarSection"
        role="alert"
      >
        <div
          class="css-1m7spai"
        >
          <div
            class="css-1vlcokg"
          >
            <span />
          </div>
          <h2
            class="css-qt9rcz-Caution"
          >
            バザーに関するお願い
          </h2>
        </div>
        <ul
          class="css-mvofyw-Caution"
        >
          <li
            class="css-u80qbw-Caution"
          >
            学内における飲食物の食べ歩きはご遠慮ください。
          </li>
          <li
            class="css-u80qbw-Caution"
          >
            学内バザーにて購入した飲食物のお持ち帰りはご遠慮ください
          </li>
          <li
            class="css-u80qbw-Caution"
          >
            各出店バザーにお並びいただく際、1つの出店につきお並びいただく人数は5人までとしておりますので、当日は係員の指示に従ってください。
          </li>
        </ul>
      </section>
      <p
        class="css-sr7z0r-BazaarSection"
      >
        飲食団体は18団体、その他レクリエーションには17団体が出店しています！ 
屋台・レク等のバザー詳細についてはこちらをクリック！
      </p>
      <a
        class="css-t2w93t-Button"
        href="/bazaar"
        role="button"
        tabindex="0"
      >
        <p
          class="css-oca8th-Button"
        >
          バザーの詳細を見る
        </p>
      </a>
    </section>
    <section
      aria-label="map"
      class="css-wkpw2c"
    >
      <h1
        class="css-4pmhej-MapSection"
      >
        Map
      </h1>
      <a
        class="css-1l4w6pd"
        href="https://res.cloudinary.com/dvgtaafq5/image/upload/q_60/f_auto/v1665765554/map_vjyaf2.webp"
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          alt="校内マップ"
          class="css-1h35ydv-MapSection"
          height="2915"
          src="/statics/map.webp"
          width="4128"
        />
      </a>
    </section>
    <section
      aria-label="sponsors"
      class="css-wkpw2c"
    >
      <h1
        class="css-zsevfz-SponsorSection"
      >
        Sponsor
      </h1>
      <p
        class="css-vrdidw-SponsorSection"
      >
        第57回鈴鹿高専祭を開催するにあたり、多くの方にご支援賜りました。
こちらのページにてご協賛いただいた企業様、ご支援いただいた個人様のお名前を記載しておりますのでぜひご覧ください。
      </p>
      <a
        class="css-1ye31l5-Button"
        href="/sponsors"
        role="button"
        tabindex="0"
      >
        <p
          class="css-oca8th-Button"
        >
          READ MORE
        </p>
      </a>
    </section>
    <section
      aria-label="お問い合わせ"
      class="css-wkpw2c"
    >
      <h1
        class="css-1461gzd-ContactSection"
      >
        Contact us
      </h1>
      <p
        class="css-25tbau-ContactSection"
      >
        第57回鈴鹿高専祭に関するお問い合わせは下のボタンからお願いいたします（google formsが開きます）
      </p>
      <a
        class="css-jabq07-Button"
        href="https://forms.gle/MU9338vfudKPEnwS7"
        rel="noopener noreferrer"
        role="button"
        tabindex="0"
        target="_blank"
      >
        <p
          class="css-7bzdgk-Button"
        >
          お問い合わせ
        </p>
      </a>
    </section>
    <section
      aria-label="鈴鹿高専へのアクセス"
      class="css-zg1vud"
    >
      <h1
        class="css-8i7hlj-AccessSection"
      >
        Access
      </h1>
      <div
        class="css-1dydy61-AccessSection"
      >
        <p
          class="css-fgh3k-AccessSection"
        >
          〒
          510-0294 三重県鈴鹿市白子町
近鉄白子駅から徒歩約30分
東旭が丘三丁目バス停から徒歩約7分
        </p>
        <iframe
          allowfullscreen=""
          class="css-vt5p73-AccessSection"
          id="map"
          loading="lazy"
          referrerpolicy="no-referrer-when-downgrade"
          role="document"
          src="https://www.google.com/maps/embed/v1/view?key=AIzaSyCk9WMilIT2wiBMqcn1YZQKJz5Lzy9ipQg&center=34.85167124241317%2C136.58132309784355&zoom=15"
          title="鈴鹿工業高等専門学校の地図"
        />
      </div>
      <div
        class="css-wkpw2c"
      >
        <h2
          class="css-1cr981d-AccessSection"
        >
          お車でお越しの方へ
        </h2>
        <p
          class="css-q76i9-AccessSection"
        >
          駐車場はスペースに限りがございますのでご了承ください。
30日(日)は鈴鹿医療科学大学 白子キャンパス 様
31日(月)はMEGAドン・キホーテUNY鈴鹿店 様の屋上駐車場　をご利用ください。
        </p>
        <img
          alt="周辺の駐車場の地図"
          class="css-oa44wn-AccessSection"
          height="464"
          loading="lazy"
          src="/statics/parking.webp"
          width="640"
        />
      </div>
    </section>
    <section
      aria-label="鈴鹿高専について"
      class="css-wkpw2c"
    >
      <h1
        class="css-9ill3l-AboutSection"
      >
        NIT,Suzuka College
      </h1>
      <p
        class="css-1x1cl0z-AboutSection"
      >
        鈴鹿高専については本校ホームページをご覧ください。
      </p>
      <a
        class="css-t2w93t-Button"
        href="https://www.suzuka-ct.ac.jp"
        rel="noopener noreferrer"
        role="button"
        tabindex="0"
        target="_brank"
      >
        <p
          class="css-oca8th-Button"
        >
          鈴鹿高専ホームページ
        </p>
      </a>
    </section>
  </main>
</div>
`;
